llmfoundry/models/hf/hf_fsdp.py

```diff
@@ -72,13 +77,17 @@ def hf_get_hidden_layers(model: PreTrainedModel):
         - transformer.h: (BloomForCausalLM, GPT2LMHeadModel, GPTJForCausalLM)
         - model.decoder.layers: (OPTForCausalLM)
         - gpt_neox.layers: (GPTNeoXForCausalLM)
+        - model.layers: (LLaMaForCausalLM)
```

Comment:
Hi! I think this got renamed to `LlamaForCausalLM` later on (different punctuation), not sure if this will have to be renamed in the future.
```
>>> transformers.__version__
'4.28.1'
>>> from transformers import LlamaForCausalLM
```